code/modules/antagonists/eldritch_cult/eldritch_knowledge.dm

```diff
@@ -25,6 +24,10 @@
 	var/list/result_atoms = list()
 	///What path is this on defaults to "Side"
 	var/route = PATH_SIDE
+	///Cast delay when using the rune
+	var/cast_time = 10	//short delay so you can cancel?
+	//Increases the total amount of followers
+	var/followers_increment = 0
```

Comment:
Those variables are not used in this PR. Are you going to use them in future PRs?